swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala

```diff
@@ -0,0 +1,261 @@
+package org.http4s
+package rho
+package swagger
+
+import org.http4s.rho.bits.HeaderAST.HeaderRule
+import org.http4s.rho.bits.PathAST._
+import org.http4s.rho.bits.QueryAST.{QueryCapture, QueryRule}
+import org.http4s.rho.bits.ResponseGenerator.EmptyRe
+import org.http4s.rho.bits._
+
+import scala.reflect.runtime.universe.Type
+
+import scalaz._, Scalaz._
+
+class SwaggerModelsBuilder(formats: SwaggerFormats) {
```

Comment:
A lot of methods on there are left public. I assume thats for the tests I've yet to scroll down too. Could they be made package private or some other constricted derivative to avoid the one in a million chance someone decides to use them directly?